src/acl/external/AD_group/ext_ad_group_acl.cc

```diff
@@ -823,10 +824,12 @@ main(int argc, char *argv[])
             DefaultDomain = xstrdup(machinedomain);
     }
     debug("%s " VERSION " " SQUID_BUILD_INFO " starting up...\n", argv[0]);
-    if (use_global)
+    if (use_global) {
         debug("Domain Global group mode enabled using '%s' as default domain.\n", DefaultDomain);
-    if (use_case_insensitive_compare)
+    }
+    if (use_case_insensitive_compare) {
         debug("Warning: running in case insensitive mode !!!\n");
+    }
```

Comment:
IMO, the two pairs of curly braces added above are unwanted from general code style point of view. Moreover, their addition is inconsistent with other **single-statement** `if`s in this file. Not adding these curly braces will keep this code intact. Please undo and check for similar additions.

Reply:
Ok. I'm leaving a few in (e.g. at line 226 and line 355), otherwise the pattern would be:

```
if () {
  // lots of code
} else
  debug(); 
```
which I find very unclear. It is unfortunately a pretty common pattern in the code, but a full refactoring is out of scope

There is also a :
```
if()
  debug(...);
else {
  lots_of_code();
}
```
that has a similar (if slightly lesser) problem

Done in cf05254f00